### PR TITLE
UI fix

### PR DESCRIPTION
**Pull request by rkapoor10**
made the side bar responsive and adjusted content to fit screen.

### CODE REVIEW

File: src/components/shared/sidemenu/sidemenu.css [hunk 1]
```diff
@@ -43,15 +45,24 @@
...
     font-size: 1.5rem;
 }
 
 
+
+
+@media only screen and (max-width:1010px){
+    /* .sidemenu-container{
+        display:none;
+    } */
```
Comment (harshitadaswani) [inline src/components/shared/sidemenu/sidemenu.css:62]: remove comments

File: src/components/shared/navbar/navbar.css [hunk 2]
```diff
@@ -104,17 +110,27 @@
...
 @media only screen and (max-width: 750px) {
   .search-btn-alone {
     display: block;
   }
-  .flex-title {
-    display: none;
-  }
+  /* .flex-title {
+    font-size: 10px;
+  } */
```
Comment (t007rushi) [inline src/components/shared/navbar/navbar.css:129]: remove comments

**Review by JantuDeb — APPROVED** [review]
Looks good to merge

File: src/components/shared/sidemenu/sidemenu.css [hunk 3]
```diff
@@ -1,12 +1,14 @@
 .sidemenu-container{
     padding-top: 1rem;
-    width:15rem;
+    /* width:15rem; */
```
Comment (t007rushi) [inline src/components/shared/sidemenu/sidemenu.css:3]: remove comments

File: src/components/shared/sidemenu/sidemenu.css [hunk 4]
```diff
@@ -1,12 +1,14 @@
...
     height:100%;
     background-color: 
     #202020;
     
 }
 
+
+
```
Comment (harshitadaswani) [inline src/components/shared/sidemenu/sidemenu.css:11]: format properly

File: src/components/shared/navbar/navbar.css (hunk 2 above)
Comment (harshitadaswani) [inline src/components/shared/navbar/navbar.css:129]: remove comments

**Review by harshitadaswani — APPROVED** [review]
remove comments

File: src/components/shared/sidemenu/sidemenu.css [hunk 5]
```diff
@@ -1,12 +1,14 @@
 .sidemenu-container{
     padding-top: 1rem;
-    width:15rem;
+    /* width:15rem; */
     height:100%;
     background-color:
```
Comment (t007rushi) [inline src/components/shared/sidemenu/sidemenu.css:5]: use variable here

**Review by t007rushi — APPROVED** [review]
recall what you are 
looks good to merge after working on some suggestions

File: src/components/shared/sidemenu/sidemenu.css (hunk 5 above)
Comment (rkapoor10) [reply src/components/shared/sidemenu/sidemenu.css:5]: done

File: src/components/shared/sidemenu/sidemenu.css (hunk 3 above)
Comment (rkapoor10) [reply src/components/shared/sidemenu/sidemenu.css:3]: done

File: src/components/shared/navbar/navbar.css (hunk 2 above)
Comment (rkapoor10) [reply src/components/shared/navbar/navbar.css:129]: done

Comment (rkapoor10) [reply src/components/shared/navbar/navbar.css:129]: done

File: src/components/shared/sidemenu/sidemenu.css (hunk 4 above)
Comment (rkapoor10) [reply src/components/shared/sidemenu/sidemenu.css:11]: done

File: src/components/shared/sidemenu/sidemenu.css (hunk 1 above)
Comment (rkapoor10) [reply src/components/shared/sidemenu/sidemenu.css:62]: done

**Pull request merged**